jdaviz/configs/imviz/plugins/links_control/links_control.py

```diff
@@ -150,6 +151,13 @@ def _link_image_data(self):
             error_on_fail=False,
             update_plugin=False)
 
+    def _check_if_many_data_with_wcs(self):
+        num_data_with_wcs = 0
+        for data in self.app.data_collection:
+            num_data_with_wcs += 1 if hasattr(data.coords, 'pixel_to_world') else 0
+
+        self.wcs_available = num_data_with_wcs > 1

```

Comment:
We don't really care after finding the second, so can we break in that case?
```suggestion
        num_data_with_wcs = 0
        for data in self.app.data_collection:
            if hasattr(data.coords, 'pixel_to_world'):
                if num_data_with_wcs:
                    self.wcs_available = True
                    break
                num_data_with_wcs += 1
        else:
            self.wcs_available = False

```